docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '2'
+services:
+  ms: &ms
+    build:
+      context: .
```

Comment:
That could be cool. what about `image: metasploit`? I like `msfconsole` but really the image is much more than the `msfconsole`. `metasploitframework` works, its just long.